spec/models/efile_error_spec.rb

```diff
@@ -29,8 +29,8 @@
   end
 
   it 'converts paths to controllers' do
-    controller = EfileError.path_to_controller("w2")
-    expect(controller).to eq StateFile::Questions::W2Controller
+    controller = EfileError.path_to_controller("name-dob")
+    expect(controller).to eq StateFile::Questions::NameDobController
```

Comment:
the name-dob page's days are limited - could you choose a different controller? "income-review" should be reliable

Reply:
oh, I didn't notice it was used in other tests too. nevermind, that can all be fixed when the page is removed